### ISSUE TITLE
Missing values for ALL history variables for ALL time-samples other than the first

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/ctsm#130]
**Erik Kluzek < erik > - 2016-02-05 13:06:42 -0700**
**Bugzilla Id:** [2281](http://bugz.cgd.ucar.edu/show_bug.cgi?id=2281)
**Bugzilla CC:**  andre,  dlawren,  dll,  oleson,  rfisher,  sacks,  tomas, 

I see this issue with the following test case in clm4_5_7_r164

SMS_D_Ld10_Mmpi-serial.CLM_USRDAT.I1PTCLM45.yellowstone_pgi.clm-USUMB

with the following change added to user_nl_clm...

 hist_fincl2 = 'TLAI'
 hist_nhtfrq = -24,-24
 hist_mfilt  = 3,3

And the problem occurs for both the h0 and h1 files. All time-samples other than the first for all history variables are set to missing value. So for example on the h1 files...

 TLAI =
  0,
  _,
  _ ;

So the workaround for now is to keep hist_mfilt at 1.

**Comment by ekluzek** [comment]
**Erik Kluzek < erik > - 2016-02-05 13:32:53 -0700**

I do NOT, get this behavior in clm4_5_6_r158. Rosie indicated she didn't see this in respmods_n01_clm4_5_6_r160. PIO2 first came in clm4_5_6_r159, with another update in clm4_5_7_r164. So I tried clm4_5_7_r163 and I do NOT see the behavior there. So it looks like this is an issue with the version of PIO2 used in cime4.3.9 (the update between r163 and r164 was to update cime).

**Comment by ekluzek** [comment]
**Erik Kluzek < erik > - 2016-02-05 14:46:09 -0700**

OK, I tried a case updated to the latest trunk tag clm4_5_7_r166 where I updated cime to cime4.4.3. The exact same case fails, (because of unrelated surface dataset issues) but I did a similar case...

SMS_D_Ld10_Mmpi-serial.1x1_brazil.ICRUCLM45.yellowstone_pgi

and that works! The default there is PIO1, but I also tried it with PIO2 and that works as well. So it looks like both PIO1 and PIO2 that's in cime4.4.3 is more robust than the version being used in the latest clm trunk tags.
